external-crates/move/crates/move-package-alt/src/schema/manifest.rs

```diff
@@ -38,6 +49,8 @@ pub struct ParsedManifest {
 pub struct PackageMetadata {
     pub name: Spanned<PackageName>,
     pub edition: String,
+    #[serde(skip)]
```

Comment:
I think I like this! It's a little bit of scope creep for the `schema` package, but I think it's useful enough to be a good choice